### ISSUE TITLE
A port for emacs ?

### ISSUE THREAD

**Issue by spcbfr** [Bugswriter/tuxi#196]
I think an emacs tuxi mode would be nice, for someone who uses emacs all day, instead of having to open vterm or eshell I can just M-x tuxi and type my question and it will return the answer in a mini buffer

**Comment by BeyondMagic** [comment]
Can't you make a script to call tuxi directly into Emacs Lisp? Isn't it faster than recreating one only in Emacs Lisp?

**Comment by spcbfr** [comment]
Yes that's what I meant, I am just to competent enough to create that
